newm-shared/src/main/kotlin/io/newm/shared/ktx/LongExt.kt

```diff
@@ -1,18 +1,25 @@
 package io.newm.shared.ktx
 
+import java.text.NumberFormat
 import java.text.SimpleDateFormat
 import java.time.LocalDateTime
 import java.time.ZoneOffset
 import java.util.Date
+import java.util.Locale
 import java.util.TimeZone
 
-private val minutesSecondsFormatter =
-    SimpleDateFormat("mm:ss").apply {
-        timeZone = TimeZone.getTimeZone("UTC")
-    }
+private val minutesSecondsFormatter = SimpleDateFormat("mm:ss").apply {
```

Comment:
Ensure thread-safety for the things in this class by wrapping stuff with `ThreadLocal`.